webclient/cypress/integration/feature/user-profiles/screen/UserProfiles.spec.ts

```diff
@@ -0,0 +1,129 @@
+/// <reference types="cypress" />
+import createUser from "../../../../fixtures/createUser.json";
+import getUsers from "../../../../fixtures/getUsers.json";
+
+context("Actions", () => {
+  before(() => {
+    cy.intercept("POST", "**/graphql", (req) => {
+      if (req.body?.query.includes("query GetUsersQuery")) {
+        req.alias = "getUsersQuery";
+        req.reply((res) => {
+          res.send({
+            body: getUsers,
+            statusCode: 200,
+          });
+        });
+      }
+
+      if (req.body?.query.includes("mutation CreateUserMutation")) {
+        req.alias = "createUserMutation";
+        req.reply((res) => {
+          res.send({
+            body: createUser,
+            statusCode: 200,
+          });
+        });
+      }
+    });
+
+    cy.intercept(
+      {
+        method: "GET",
+        url: "https://api.unsplash.com/photos/random",
+      },
+      {
+        statusCode: 200,
+        fixture: "unsplash.json",
+      }
+    );
+
+    cy.intercept(
+      {
+        method: "GET",
+        url: "https://api.mapbox.com/",
+      },
+      {
+        statusCode: 200,
+      }
+    );
+
+    cy.visit("http://localhost:3000");
+    cy.injectAxe();
+  });
+
+  it("create a new user", () => {
+    const name = "Porfirio Jimenez";
+    const address = "An address";
+    const description = "A description";
+    const dob = "2021-01-01";
+
+    cy.get("[data-testid=users-list-create-user-card] button").click();
+    cy.checkA11y(null, null, terminalLog);
+
+    cy.get("[data-testid=create-user-card-name-input]").type(name).should("have.value", name);
+    cy.get("[data-testid=create-user-modal-address-input]")
+      .type(address)
+      .should("have.value", address);
+    cy.get("[data-testid=create-user-modal-description-input]")
+      .type(description)
+      .should("have.value", description);
+    cy.get("[data-testid=create-user-modal-dob-input]").type(dob).should("have.value", dob);
```

Comment:
Looks like we test cypress - we don't have to test `.should("have.value", dob)`, we test the end result - for example was user created in API and listed on page after adding it + did success toast show up etc.